docs/includes/apiargs-MongoDBCollection-method-dropIndex-param.yaml

```diff
@@ -1,10 +1,10 @@
 arg_name: param
 name: $indexName
-type: string
+type: string| :phpclass:`MongoDB\\Model\\IndexInfo`
```

Comment:
Note: I added a space after the `|` so the `:phpclass:` renders properly in the docs.

Reply:
Interesting. Can you ask someone in #docs about this later? Just curious if they have a work-around for it.